{
	"class": "vulnerability",
	"id": "oval:com.ubuntu.xenial:def:2024452310000000",
	"version": "1",
	"metadata": {
		"title": "CVE-2024-45231 on Ubuntu Pro FIPS 16.04 LTS (xenial) - low",
		"description": "Due to unhandled email sending failures, thedjango.contrib.auth.forms.PasswordResetForm class allowed remote attackers toenumerate user emails by issuing password reset requests and observing theoutcomes.To mitigate this risk, exceptions occurring during password reset email sendingare now handled and logged using the \"django.contrib.auth\" logger.",
		"affected": {
			"family": "unix",
			"platform": "Ubuntu Pro FIPS 16.04 LTS"
		},
		"reference": {
			"source": "CVE",
			"ref_id": "CVE-2024-45231",
			"ref_url": "https://www.cve.org/CVERecord?id=CVE-2024-45231"
		},
		"advisory": {
			"severity": "Low",
			"rights": "Copyright (C) 2024 Canonical Ltd.",
			"public_date": "2024-09-03",
			"cve": {
				"text": "CVE-2024-45231",
				"href": "https://ubuntu.com/security/CVE-2024-45231",
				"public": "20240903"
			},
			"public_date_at_usn": "2024-09-03",
			"assigned_to": "leosilva",
			"ref": [
				"https://ubuntu.com/security/notices/USN-6987-1"
			]
		}
	},
	"notes": {},
	"criteria": {
		"criterias": [
			{
				"operator": "OR",
				"criterions": [
					{
						"test_ref": "oval:com.ubuntu.xenial:tst:2019123080000000",
						"comment": "python-django package in xenial is affected and may need fixing."
					}
				]
			}
		]
	}
}
